src/Substructure.jl

```diff
@@ -0,0 +1,285 @@
+"""
+    has_parents(p, clusterseq) -> (boolean, Int64, Int64)
+
+Checks if the jet `p` is a child of two other jets, after clustering 
+
+# Arguments
+- `p`: The jet to check.
```

Comment:
A small documentation point: the docstring should match the argument in the signature, including type. i.e., here it should be `p:: PseudoJet`.